frontend/src/locales/en.yml

```
@@ -217,6 +217,20 @@ nav:
   loyalUsers: Loyal users
   myOrders: My orders
   settings: Settings
+  activity: Activity
+
+activity:
```

YAMLlint (1.38.0)

[error] duplication of key "activity" in mapping

(key-duplicates)